README.md

```diff
@@ -246,6 +246,22 @@ req.checkBody({
 });
 ```
 
+You can also define location against to validate specific schema validator by adding `in` parameter as fallowing:
```

Comment:
Change to:
You can also define a specific location to validate against in the schema by adding `in` parameter as shown below: